frontend/components/wallet/WalletTable.stories.js

```diff
@@ -0,0 +1,31 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import { action } from '@storybook/addon-actions';
+import WalletTable from '~/components/wallet/WalletTable';
+
+const common = {
+    components: { WalletTable },
+    data: () => ({
+        walletAddress: faker.finance.ethereumAddress(),
+        walletETH: new BigNumber(faker.finance.amount()),
+        walletDAI: new BigNumber(faker.finance.amount()),
+        walletVatDAI: new BigNumber(faker.finance.amount()),
+        walletLastUpdatedDate: faker.date.recent(),
+    }),
+    methods: {
+        refresh: action('refresh'),
+    },
+};
+
+storiesOf('Wallet/WalletTable', module)
+    .add('Default', () => ({
+        ...common,
+        template:
+            '<WalletTable :walletAddress="walletAddress" :walletETH="walletETH" :walletDAI="walletDAI" :walletVatDAI="walletVatDAI" :walletLastUpdatedDate="walletLastUpdatedDate" @refresh="refresh" />',
```

Comment:
Did you know that you can v-bind an object pass all of it's keys as props?
```suggestion
            '<WalletTable v-bind="$data" @refresh="refresh" />',
```

Reply:
Cool. I did not know this!